internal/component/prometheus/exporter/windows/config.go

```diff
@@ -135,6 +135,7 @@ func (t SMTPConfig) Convert() windows_integration.SMTPConfig {
 
 // ServiceConfig handles settings for the windows_exporter service collector
 type ServiceConfig struct {
+	//TODO: Should we make use_api and enable_v2_collector bool in Alloy v2?
```

Comment:
Both use_api and enable_v2_collector are removed in 0.29 anyways.

Reply:
Thanks for letting me know, I removed this comment.